doc/classes/String.xml

```diff
@@ -535,12 +587,31 @@
 			<param index="2" name="capitalize_hex" type="bool" default="false" />
 			<description>
 				Converts a signed [int] to a string representation of a number.
+				Converts a signed [int] to a string representation of a decimal number, or other [param base].
+				Returns a [String] representing the given signed [int] as decimal number, or other [param base].
+				Converts the given [param number] to a string representation, with the given [param base].
```

Comment:
Huh, why is this documented four times? :thinking:

Anyway my favorite is "Converts the given [param number] to a string representation, with the given [param base]."

Reply:
Indeed, the intention was reiterate over the same sentence said different times to see which one sticks.